deployment/network-operator/charts/sriov-network-operator/crds/sriovnetwork.openshift.io_sriovnetworknodepolicies_crd.yaml

```diff
@@ -0,0 +1,112 @@
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
probably I don't understand helm (maybe you can show me a demo) but  why do we need all the crds from SR-IOV operator?

Reply:
It is the intention of this chart to deploy all elements needed to use sriov network operator in a k8s cluster, so it includes its crds.

lets have an offline talk on this